Makefile

```diff
@@ -86,15 +86,12 @@ PINNED_DEPENDENCIES := \
 	github.com/go-sql-driver/mysql@v1.5.0 \
 	github.com/urfave/cli/v2@v2.4.0
 
-# Code coverage output files.
+# Code coverage & test report output files.
 COVER_ROOT                      := ./.coverage
-UNIT_COVER_PROFILE              := $(COVER_ROOT)/unit_coverprofile.out
-INTEGRATION_COVER_PROFILE       := $(COVER_ROOT)/integration_coverprofile.out
-DB_TOOL_COVER_PROFILE           := $(COVER_ROOT)/db_tool_coverprofile.out
-FUNCTIONAL_COVER_PROFILE        := $(COVER_ROOT)/functional_$(PERSISTENCE_DRIVER)_coverprofile.out
-FUNCTIONAL_XDC_COVER_PROFILE    := $(COVER_ROOT)/functional_xdc_$(PERSISTENCE_DRIVER)_coverprofile.out
-FUNCTIONAL_NDC_COVER_PROFILE    := $(COVER_ROOT)/functional_ndc_$(PERSISTENCE_DRIVER)_coverprofile.out
+NEW_COVER_PROFILE 							:= $(COVER_ROOT)/$(shell xxd -p -l 16 /dev/urandom)_coverprofile.out # generates a new filename each time it's substituted.
```

Comment:
this generates one per `make` invocation.. you have to use `=` instead of `:=` to get one per substitution. in general it'll run just one target per invocation so it's fine, but I'd change to `=` just to be sure.

(btw, someone may ask why not use `uuidgen` or something simpler. turns out the alpine-based builder images don't have that. `xxd` isn't quite universal but it is present in those images. `od` would be even better but the version in alpine didn't support enough flags to work quite right.)